src/client/Microsoft.Identity.Client/Platforms/Features/DefaultOSBrowser/HttpListenerInterceptor.cs

```diff
@@ -89,6 +89,13 @@ public HttpListenerInterceptor(ICoreLogger logger)
                 cancellationToken.ThrowIfCancellationRequested();
 
                 // if cancellation was not requested, propagate original ex
+                if (ex is HttpListenerException)
+                {
+                    throw new Exception("HttpListenerException occurred when listening for the response from the browser. " + 
```

Comment:
Alternatively, instead of throwing, I can just log error here.

Reply:
No, let's rethrow an MsalClientException and set this exception as inner exception.

Reply:
Done.